tests/yaml_file_test.py

```diff
@@ -148,3 +149,19 @@ def test_should_skip_for_min_flash_positive(testcase, platform):
     testcase.min_flash = 200
     platform.flash = 300
     assert should_skip_for_min_flash(testcase, platform) is False
+
+
+def test_if_join_stringas_returns_joined_strings():
```

Comment:
`_join_stringas` -> `_join_strings`? I guess its just a typo but might be worth of fixing